javascripts/govuk/multivariate-test.js

```diff
@@ -77,7 +80,17 @@
       cohort, 
       2 // session level
     ); 
-    // Fire off a dummy event to set the custom var on the page.
+  };
+
+  MultivariateTest.prototype.setUpContentExperiment = function(cohort) {
+    if(this['contentExperimentId'] && this['cohorts'][cohort]['variantId']){
```

Comment:
Using dot notation here might make the method easier to read, eg `this.contentExperimentId`.

Both `this['contentExperimentId']` and `this['cohorts'][cohort]['variantId']` are used twice, they could be declared as local variables which would also improve readability.